data/companies.yml

```diff
@@ -221,10 +221,13 @@ integrations:
       - "Space Cloud integrates with Kubernetes and Istio natively to bring to you a highly scalable Serverless Platform. It encrypts all traffic by default and lets you describe communication policies to protect your microservices."
 
 pro_services:
-  - name: "Your Name Here"
-    logo: "/logos/istio.svg"
-    url: "https://istio.io/"
-    description: "Submit a PR to add your company here!"
+  - name: "CloudZone"
+    logo: "/logos/cloudzone.svg"
+    url: "https://cloudzone.io/"
+    description:
+      - "At CloudZone, we’ve been actively involved with Kubernetes and Istio since the very first editions came out." 
+      - "In today’s fast-paced, ever-changing world, we continue to research the field keeping up to date so that we can bring you the very latest thinking and solutions, whether we are supporting you through the provision of consulting, design, training or production support services."
+      - "Our certified Kubernetes administrators and cloud experts are highly skilled in building production-grade Kubernetes systems with the latest Cloud native tools for automating application deployment, scaling and management, together with the open source Istio service mesh for connecting, securing, controlling and observing services."
   - name: "Your Name Here"
```

Comment:
Please add a single line "description" and change the heading to "details".  

(See https://github.com/istio/istio.io/pull/10024/files for the fixed template, with my apologies)

Reply:
I've added "details"